index.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,900,700' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
```

Comment:
OK, What I'm looking at is this:
Lato - headings and question typography
Some kind of handwriting font (currently trying Architects Daughter) - margin comments and label
Merriweather - book font